.github/workflows/e2e-pr-test.yml

```
@@ -0,0 +1,290 @@
+  pr-comment-parser:
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'issue_comment' && github.event.issue.pull_request) || github.event_name == 'pull_request_target'
+    outputs:
+      test-type: ${{ steps.parse-comment.outputs.test-type }}
+      should-run: ${{ steps.parse-comment.outputs.should-run }}
+      pr-number: ${{ steps.parse-comment.outputs.pr-number }}
+      test-sets: ${{ steps.parse-comment.outputs.test-sets }}
+      tag-name: ${{ steps.parse-comment.outputs.tag-name }}
+    steps:
+      - name: Debug event type
+        run: |
```

actionlint (1.7.7)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)